src/app/routes/dashboard/dashboard.component.ts

```diff
@@ -151,6 +158,12 @@ export class DashboardComponent implements AfterViewInit {
     this.route.navigate(['assets']);
   }
 
+  goToVerifiablePresentation(oob: string) {
+    this.route.navigate(['vp'], {
```

Comment:
When you rebase or merge develop into this branch, then there is `RouterConst` object. It would be good to use it instead of string.